nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -77,15 +77,14 @@
    * create an error description for a nullability warning
    *
    * @param errorMessage the error message object.
-   * @param path the TreePath to the error location. Used to compute a suggested fix at the
-   *     enclosing method for the error location
    * @param descriptionBuilder the description builder for the error.
+   * @param state the visitor state (used for e.g. suppression finding).
    * @return the error description
    */
   Description createErrorDescription(
-      ErrorMessage errorMessage, TreePath path, Description.Builder descriptionBuilder) {
-    Tree enclosingSuppressTree = suppressibleNode(path);
-    return createErrorDescription(errorMessage, enclosingSuppressTree, descriptionBuilder);
+      ErrorMessage errorMessage, Description.Builder descriptionBuilder, VisitorState state) {
```

Comment:
@lazaroclapp I'm not quite seeing why we need to pass in the `VisitorState` here?  All I can see is calls to `state.getPath()`, and we were passing a `TreePath` before.  Can you point me to where we need something else from `state`?

Reply:
We don't, but there is also no reason not to. Basically, I changed the signature of the other `createErrorDescription(...)` below first, using `VisitorState` since that's already passed to other methods in this file. Given that, having both take `VisitorState` seemed more consistent. No issue with having both get `TreePath`, though if there is an issue regarding passing `VisitorState`

Reply:
That's fine.  Switching is good since it'll give more flexibility in the future if needed